### ISSUE TITLE
Adjust the design for light mode docs

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#5140]
### Describe the issue

So far we only have this design in Figma for docs:

![image](https://github.com/user-attachments/assets/c0e1deba-0686-4eec-ab9b-9e46996eff36)

There are many components for which design is missing -- this bites us in implementing in-app docs:

![image](https://github.com/user-attachments/assets/6f6a255c-bca2-44d6-91ec-a0486633f7a9)

![image](https://github.com/user-attachments/assets/b3188d41-d308-4082-b88f-037f11616f4d)

In short, @Nico-Kestra let's browse the docs on the OSS preview and add missing designs for components that don't render well in the light mode UI -- I'll show you via Huddle once @brian-mulier-p merges the in-app docs https://github.com/kestra-io/kestra-ee/issues/1535 

### Environment

- Kestra Version:
- Operating System (OS/Docker/Kubernetes):
- Java Version (if you don't run kestra in Docker):

**Comment by anna-geller** [comment]
@brian-mulier-p Nico already added design here https://www.figma.com/design/ew0uXk0NRXJ2NBBJTNe2n1/01_UI?node-id=3134-40835&t=QzYLOqhGUV6hEqCb-4

**Comment by anna-geller** [comment]
removing Nico from assignees as so far everything is in Figma:
1. design for new links
2. info/warning boxes
3. inline code blocks
4. full code blocks
5. hover over URL fields
6. color for section headers

**Comment by anna-geller** [comment]
@brian-mulier-p only a few things left, can you finalize this so we can close the subject? 🚀 

- [x] inline code blocks have bad color
![image](https://github.com/user-attachments/assets/7a187b8c-e8de-42fd-8ccf-cf99be8d7b6c)
![image](https://github.com/user-attachments/assets/86c0c2bd-5357-4c16-859c-5e623048bd56)


- [x] full code blocks lack indentation
![image](https://github.com/user-attachments/assets/f8517a6b-b5e7-42fc-bb27-43468c30400b)

- [x] design for new links
- [x] info/warning boxes

**Comment by brian-mulier-p** [comment]
final step done [here](https://github.com/kestra-io/kestra/commit/85c9892ff9d10458503b6dd93173bf1698982e9d), feel free to reopen if I missed something
